src/lib/expansion/expansion.md

```diff
@@ -0,0 +1,81 @@
+`<md-expansion-panel>` provides a way to show and hide lightweight content, by collapsing and expanding a view with a nice animation.
+
+<!-- example(expansion-overview) -->
+
+The `expanded` input allows to choose among the collapsed or expanded state. 
```

Comment:
I would omit this bit about the `expanded` property because it will be captured via API docs and examples.